### PR TITLE
Passthrough cross environment variables by default.

### PR DESCRIPTION
**Pull request by Alexhuszagh**

### CODE REVIEW

**Comment by Alexhuszagh** [comment]
bors try --target x86_64-unknown-linux-gnu

This passes through any cross configuration environment variables to the container, allowing custom configurations in the future to be forwarded to the container. An initial example is https://github.com/cross-rs/cross-toolchains/pull/15, however, this also applies to `CROSS_DEBUG` and allows us to deny known environment variables cross uses (such as `CROSS_RUNNER` and `CROSS_RUSTC_MINOR_VERSION`).

File: .changes/1073.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "type": "added",
+    "description": "allow passthrough of cross environment variables"
```
Comment (Emilgardis) [inline .changes/1073.json:3]: this makes it seems as if this was somehow not possible before

**Review by Emilgardis — APPROVED** [review]
Lgtm!

**Comment by Alexhuszagh** [comment]
bors r=Emilgardis

**Pull request merged**